controllers/metallb_controller.go

```diff
@@ -61,15 +61,25 @@ var MetalLBChartPath = MetalLBChartPathController
 // +kubebuilder:rbac:groups=monitoring.coreos.com,resources=podmonitors,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=monitoring.coreos.com,resources=servicemonitors,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups="",namespace=metallb-system,resources=services,verbs=create;delete;get;update;patch
+// +kubebuilder:rbac:groups="",namespace=metallb-system,resources=events,verbs=create;patch
+// +kubebuilder:rbac:groups="",namespace=metallb-system,resources=configmaps,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=coordination.k8s.io,namespace=metallb-system,resources=leases,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=coordination.k8s.io,namespace=metallb-system,resources=leases,verbs=get;list;watch;create;update;patch;delete
 
 // Cluster Scoped
+// +kubebuilder:rbac:groups=apps,resources=deployments;daemonsets,verbs=get;list;watch;create;update;patch;delete
```

Comment:
why these are cluster scoped? They should be namespace scoped I think (and we have them already above)? 
Same for the additions below. Cluster scope means we are giving the operator permissions to modify any daemonset in the cluster iirc, while we should keep the permissions as low as we can

Reply:
Had a typo in the role binding. fixed and removed all cluster permissions.